### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by julerobb1** [AveYo/MediaCreationTool.bat#90] [General]
This is my second time using this script, and I have to say, you did a great job. Although, I would love to know how you made it auto ciick and things.

**Comment by AveYo** [comment]
Thanks.
I just use Windows.Automation.AutomationElement .net class via powershell.
You wont find many other projects doing it successfully 
_(at least I did not find any, so I made everything following microsoft docs)_
so you're just gonna have to inspect the code.
Defined at line 443 `#:: load ui automation library`
and use at line 466 `#:: using automation to click gui buttons directly`
